src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java

```diff
@@ -64,22 +74,33 @@ private String getUrlPrefix(String wikiName) {
   }
 
   private String getUrlSuffix(String wikiName) {
+    if (env.contains("dev") && isMercuryBrowser()) {
+      return String.format(XIPIO_ADDRESS_FORMAT, XIPIO_DEFAULT_DOMAIN, XIPIO_DEFAULT_PORT);
```

Comment:
The comment is a suggestion since I wrote to you LGTM before. And you are right from my point of view it's over-engineering :wink: